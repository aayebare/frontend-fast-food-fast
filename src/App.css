.App {
    margin: 0;
    font-family: Arial, Helvetica, sans-serif;
    height: 850px;
    background: url('https://images.unsplash.com/photo-1515003197210-e0cd71810b5f?ixlib=rb-1.2.1&ixid=eyJhcHBfaWQiOjEyMDd9&auto=format&fit=crop&w=1950&q=80') fixed;
  
  }