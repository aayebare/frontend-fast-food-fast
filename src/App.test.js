import React from 'react';
import App from './App';
import { shallow } from 'enzyme';


describe('renders without crashing', () => {
  const wrapper  = shallow(<App />)
  it('it renders the app component', () => {
    expect(wrapper).toMatchSnapshot()
  });
});
