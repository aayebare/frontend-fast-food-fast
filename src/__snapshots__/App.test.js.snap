// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`renders without crashing it renders the app component 1`] = `ShallowWrapper {}`;
